config/manifests/bases/dynatrace-monitoring.clusterserviceversion.yaml

```diff
@@ -0,0 +1,326 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Deep Insights
+    categories: Monitoring,Logging & Tracing,OpenShift Optional
+    containerImage: registry.connect.redhat.com/dynatrace/dynatrace-oneagent-operator
```

Comment:
Kubernetes and OpenShift have different images.

Reply:
I know but for the base you can only provide one of them

Reply:
Two different bases then? Can be looked at later.